net/tayga/src/opnsense/mvc/app/controllers/OPNsense/Tayga/forms/general.xml

```diff
@@ -11,6 +11,12 @@
         <type>text</type>
         <help>This is not your router's IPv4 address. Tayga requires its own address because it acts as an IPv4 and IPv6 router, and needs to be able to send ICMP messages. This address can safely be located inside the dynamic-pool prefix.</help>
     </field>
+    <field>
+        <id>general.v4destination</id>
+        <label>IPv4 Route Target</label>
+        <type>text</type>
+        <help>Set a random unused IP address which will be used to route traffic via nat64 interface.</help>
```

Comment:
This might all be a little misleading. This is the address of the nat64 TUN interface, so something like `v4tunaddress` and 'NAT64 interface address' might be clearer. 
Also, we need an IPv6 address for the TUN interface, too (`v6tunaddress`?).